packages/rexray/buildinfo.json

```diff
@@ -1,9 +1,9 @@
 {
-  "docker": "golang:1.7",
+  "docker": "golang:1.9",
   "single_source" : {
     "kind": "git",
-    "git": "https://github.com/codedellemc/rexray.git",
-    "ref": "2a7458dd90a79c673463e14094377baf9fc8695e",
-    "ref_origin": "v0.9.0"
+    "git": "https://github.com/rexray/rexray.git",
+    "ref": "fc8bfbd2d02c2690fc3a755a9560dd12c88e0852",
```

Comment:
Do we need to bump the version of rexray?

Reply:
Yes, we need to bump it to pull in https://github.com/rexray/rexray/pull/995. That fix is first available in v0.10, but I bumped to v0.11.2 since it's a more recent release.

Reply:
ah, ic. have you tested the new version, like manually?

Reply:
I haven't, I was relying on this integration test: https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/test_rexray.py#L16

Do you have a test plan you run through for REX-Ray changes?

Reply:
Discussed offline w/ @branden and manual testing seems to indicate things are working about as well as can be expected.